src/parseOption.ts

```diff
@@ -132,7 +143,9 @@ export function prepareAllFieldsData(itemData: ScenegraphData, options: Option =
   const fieldOptions: TemporaryFieldOption = {};
   if (options && options.fields) {
     for (const optionField of options.fields) {
-      fieldOptions[optionField.field] = optionField;
+      const fieldStr = isFunction(optionField.field) ?
+        optionField.field(itemData) : optionField.field;
```

Comment:
I think we might not have to change anything in `prepareAllFieldsData` since it is used for when `showAllFields` is true, which implies that we should not use any `field` to populate the tooltip

Reply:
@sirahd that's what I thought at first too, but `options.fields` is still accepted when `showAllFields` is true, and this function uses `field` to generate a key in the `fieldOptions: TemporaryFieldOption` map.

So, it seems we should keep this here to be safe, so that nothing breaks if someone passes a function for `field` when using `showAllFields`.

Reply:
Nevermind -- by separating `valueAccessor` out from `field`, this definitely doesn't need to change. Will revert.

Reply:
Actually, this code is no longer necessary now that I'm leaving `field` alone and instead adding `valueAccessor`. Will remove.